docs/template_plugin/tests/functional/op_reference/region_yolo.cpp

```diff
@@ -0,0 +1,155 @@
+// Copyright (C) 2021 Intel Corporation
```

Comment:
@wilson-seok Please update copy right in next PR

Reply:
@ilyachur Got it! I will use 2022 of copy right in future!